daml-lf/spec/daml-lf-1.rst

```diff
@@ -2540,6 +2690,8 @@ ContractId functions
   Returns ``'True'`` if the first contact id is equal to the second,
   ``'False'`` otherwise.
 
+  [*Available in version < 1.dev*]
```

Comment:
This isn’t true is it? We only added this in 1.5 or 1.6 iirc

Reply:
I seems it was there from the beginning. At least it is there since the open sourcing. 
also in the proto it is mapped to 87 which is lower that `EQUAL_LIST` and `TRACE`.